src/main/java/com/yelp/nrtsearch/server/luceneserver/search/SearchRequestProcessor.java

```diff
@@ -149,13 +151,6 @@ public static SearchContext buildContextForRequest(
       profileResult.setRewrittenQuery(query.toString());
     }
 
-    if (searchRequest.getFacetsCount() > 0) {
```

Comment:
Is this not required any more?

Reply:
This wrapping now happens in `SearchHandler` after potentially building a hybrid vector query.